php/src/Google/Protobuf/Internal/Message.php

```diff
@@ -61,19 +61,27 @@ class Message
     /**
      * @ignore
      */
-    public function __construct($desc = NULL)
+    public function __construct($data = NULL)
     {
-        // MapEntry message is shared by all types of map fields, whose
```

Comment:
These comments should be kept somewhere.

Reply:
Added  comments back